### DISCUSSION TITLE
What is the farthest the Doomsday Clock has been?

### DISCUSSION THREAD

**Discussion by SANKARAMDAS** [ParanoidUser/yolo#755] [Galaxy Trivia] [answered]
* When did the clock's original setting?

**Comment by peterbaccaro** [comment, marked as answer]
The Doomsday Clock is a symbolic representation of how close humanity is to a global catastrophe, particularly a nuclear war. It was created in 1947 by the Bulletin of the Atomic Scientists, and its time is set annually by a group of experts from various fields. The closeness to midnight represents the metaphorical "doomsday".
The farthest it has ever been from midnight was **17 minutes in 1991**, following the end of the Cold War and the signing of the Strategic Arms Reduction Treaty (START) between the US and the Soviet Union.
The original setting of the Doomsday Clock was in 1947, when it was first created by the Bulletin of the Atomic Scientists. At that time, the clock was set at **7 minutes to midnight**.

**Reply by ParanoidUser** [reply]
*Why did the Doomsday Clock go to the gym?
Because it wanted to work on its "countdowns"!* 🕛🙄

**Reply by peterbaccaro** [reply]
😆 ...I thought it was because it wanted to make sure it was in shape to outrun any impending global disasters.
